### ISSUE TITLE
Migrations not happening automatically in production

### ISSUE THREAD

**Issue by mgilbertnz** [flyway/flyway-play#67]
Plugins
```addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.0")```

build.sbt
```libraryDependencies += "org.flywaydb" %% "flyway-play" % "5.3.1"```

Running play application in Dev mode, migrations are prompted as expected.
Running same application in Prod mode, doesn't run migration scripts.

production.conf
```
play.modules.enabled += "org.flywaydb.play.PlayModule"
db.default.migration.initOnMigrate=true
db.default.migration.auto=true
```

Starting the server with:
`./server -Dconfig.resource=production.conf`

Startup logging:
```
[info] p.a.d.DefaultDBApi - Database [default] initialized at jdbc:postgresql://localhost/toprail
[info] application - Creating Pool for datasource 'default'
[info] play.api.Play - Application started (Prod) (no global state)
[info] p.c.s.AkkaHttpServer - Listening for HTTP on /0:0:0:0:0:0:0:0:9000
[error] c.s.h.t.d.PrintlnRequestLogger - org.postgresql.util.PSQLException: ERROR: relation "users" does not exist
```

Users table "should" have been created with migration scripts however they are not run.

This is happening on a blank database so it needs to init the flyway_schema_history table.

Any help much appreciated.

**Comment by tototoshi** [comment]
Published 5.3.2-SNAPSHOT. Could you try this? 
(Thank you @jopecko !)

https://oss.sonatype.org/content/repositories/snapshots/org/flywaydb/flyway-play_2.12/5.3.2-SNAPSHOT/

**Comment by Rogach** [comment]
I had the same problem, and upgrading to `5.3.2-SNAPSHOT` fixed the problem.

When can we expect a new release of the library with this fix?

**Comment by tototoshi** [comment]
Thank you for your feedback. Published 5.3.2 
https://repo1.maven.org/maven2/org/flywaydb/flyway-play_2.12/5.3.2/
